packages/theme/stories/characteristics/overlayable.stories.mdx

```diff
@@ -14,13 +14,13 @@ Components that may form a hierarchy and stack upon or beneath other elements of
 
 ### Shadow (Elevation)
 
-Scroll: Dropshadow found at the top of an overflowing scrollable area (e.g. Comments)
-Border region: Dropshadow found around application regions (e.g. App Header, Drawer)
-Default: Dropshadow default (e.g. Card)
-Hover: Dropshadow found around a component using the default (e.g. Card) on hover state
-Popout: Dropshadow found around temporarily overlaying elements (e.g. Tooltip)
-Drag: Dropshadow found around draggable objects on drag (e.g. Tabs)
-Modal: Dropshadow found around modal windows (e.g. Dialog)
+* Scroll - Dropshadow found at the top of an overflowing scrollable area (e.g. Comments)
```

Comment:
Maybe, a table will be clearer here?

Reply:
I think a bulleted list is fine, but a table would also work